Changes

```diff
@@ -1,6 +1,26 @@
 Release history for Zonemaster component Zonemaster-Backend
 
 
+v11.2.0 2024-07-01 (part of Zonemaster v2024.1 release)
+
+ [Release information]
+ - Translations have not been fully updated in this release. They will
+   be updated in an upcoming release.
+ - Migration of the database is required by this release to retain
+   consistency in results in the database.
+
+ [Features]
+ - Ensures Backend’s messages get translated (#1166)
```

Comment:
Bringing in line with the appropriate PR’s title (see #1166):
```suggestion
 - Ensure Backend’s messages stay translated (#1166)
```

Reply:
The bullet always starts with a verb in passive form. The changes "ensures ...".

Reply:
Doesn't "get translated" cover that? Nothing stays translated.

Reply:
"are translated"?

Reply:
In general, unit tests are not covered by the Changes file. We agreed on that some years back. Travis changes are not included either.

Reply:
In that case, how about:
```suggestion
 - Fixes regression that caused Backend’s messages not to be translated (#1166)
```

Reply:
Fixed.